### PR TITLE
Add guide to quarkus-extension.yaml

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: docs/modules/ROOT/pages/includes/attributes.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:quarkus-version: 2.5.1.Final
+:quarkus-version: 2.6.0.Final
```
Comment (gastaldi) [inline docs/modules/ROOT/pages/includes/attributes.adoc:1]: Can we get this info from Maven?

**Review by gastaldi — APPROVED** [review]

File: runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 2]
```diff
@@ -6,4 +6,5 @@ metadata:
   - "git"
   categories:
   - "miscellaneous"
-  status: "stable"
\ No newline at end of file
+  guide: https://quarkiverse.github.io/quarkiverse-docs/quarkus-tika/dev/index.html
```
Comment (gsmet) [inline runtime/src/main/resources/META-INF/quarkus-extension.yaml:9]: raaah the URL in my browser address wasn't refreshed, let me fix that

File: docs/modules/ROOT/pages/includes/attributes.adoc (hunk 1 above)
Comment (gsmet) [reply docs/modules/ROOT/pages/includes/attributes.adoc:1]: It's coming from Maven during the build. The problem is that you need it defined in attributes for Antora to pick them up.

It's fixed when releasing anyway as the release process updates it.

File: runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 3]
```diff
@@ -6,4 +6,5 @@ metadata:
   - "git"
   categories:
   - "miscellaneous"
-  status: "stable"
\ No newline at end of file
+  guide: https://quarkiverse.github.io/quarkiverse-docs/quarkus-jgit/dev/index.html
```
Comment (gsmet) [inline runtime/src/main/resources/META-INF/quarkus-extension.yaml:9]: Fixed!

File: docs/modules/ROOT/pages/includes/attributes.adoc (hunk 1 above)
Comment (gastaldi) [reply docs/modules/ROOT/pages/includes/attributes.adoc:1]: The problem is that this needs to be manually updated when dependabot opens a PR updating quarkus

Comment (gsmet) [reply docs/modules/ROOT/pages/includes/attributes.adoc:1]: Yes, I understand that. Be my guest for finding a better solution :). We could trigger an additional commit automatically but given it's naturally fixed on release anyway, I don't see it as a major problem and I decided to spend my time elsewhere as the situation is good enough (TM) for me :).

Comment (gastaldi) [reply docs/modules/ROOT/pages/includes/attributes.adoc:1]: Sure, I'll play around with maven-resources-plugin and the filtering mechanism here, but I agree this is good enough for now

**Pull request merged**